pkg/dao/repository_configs.go

```diff
@@ -694,6 +722,23 @@ func (r repositoryConfigDaoImpl) Update(ctx context.Context, orgID, uuid string,
 	return updatedUrl, nil
 }
 
+func (r repositoryConfigDaoImpl) InternalOnly_ResetFailedSnapshotCount(ctx context.Context, rcUuid string) error {
+	res := r.db.WithContext(ctx).Model(models.RepositoryConfiguration{}).Where("uuid = ?", rcUuid).UpdateColumn("failed_snapshot_count", 0)
+	if res.Error != nil {
+		return fmt.Errorf("Failed to update failed_snapshot_count: %w", res.Error)
+	}
+	return nil
+}
+
+func (r repositoryConfigDaoImpl) InternalOnly_IncrementFailedSnapshotCount(ctx context.Context, rcUuid string) error {
+	res := r.db.WithContext(ctx).Exec("UPDATE repository_configurations SET failed_snapshot_count = failed_snapshot_count + 1  WHERE uuid = ? AND repository_configurations.deleted_at IS NULL", rcUuid)
+	// res := r.db.WithContext(ctx).Model(models.RepositoryConfiguration{}).Where("uuid = ?", rcUuid).UpdateColumn("failed_snapshot_count", "failed_snapshot_count + 1")
```

Comment:
```suggestion
```
this probably wasn't meant to commited 🔍😅

Reply:
IQE API docs updated

Reply:
fixed!